solutions/ha-nva/layer-7-ingress/README.md

```diff
@@ -0,0 +1,106 @@
+[![Deploy To Azure US Gov](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.svg?sanitize=true)](https://portal.azure.us/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmspnp%2Fsamples%2Fmaster%2Fsolutions%2Fha-nva%2Flayer-7-ingress%2Fazuredeploy.json)
+[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2Fmspnp%2Fsamples%2Fmaster%2Fsolutions%2Fha-nva%2Flayer-7-ingress%2Fazuredeploy.json)
+
+#### Azure CLI
+
+Use the following command to create a resource group for the deployment.
+
+```bash
+az group create --name ha-nva-l7i --location eastus
+```
+
+Run the following command to deploy the Layer 7 Ingress example architecture.
+
+```bash
+az deployment group create --resource-group ha-nva-l7i \
+    --template-uri https://raw.githubusercontent.com/mspnp/samples/master/solutions/ha-nva/layer-7-ingress/azuredeploy.json
+```
+
+#### PowerShell
+
+Use the following command to create a resource group for the deployment.
+
+```powershell
+New-AzResourceGroup -Name ha-nva-l7i -Location eastus
+```
+
+Run the following command to deploy the Layer 7 Ingress example architecture.
+
+```powershell
+New-AzResourceGroupDeployment -ResourceGroupName ha-nva-l7i `
+    -TemplateUri https://raw.githubusercontent.com/mspnp/samples/master/solutions/ha-nva/layer-7-ingress/azuredeploy.json
+```
+
+#### Redeploying the architecture to the same Resource Group or Resource Group name
+
+You cannot redeploy this template to the same Resource Group without first deleting the NVA and web VM resources. The deployment script that generates the SSH key will generate a new key with each deployment and Azure does not allow redeployment of VMs that change the `linuxConfiguration.ssh.publicKeys` property.
+
+If you are redeploying the template using the same Resource Group name, you will need to purge the previously created Azure Key Vault.  Azure requires globally unique names for Key Vault resources and this template will generate a unique name based on your subscription and Resource Group. Additionally Key Vault implements a soft-delete feature which is enabled by default. If you want to deploy the template to a Resource Group of the same name, you will have to purge the previous instance of the Key Vault following the instructions at [List, recover, or purge a soft-deleted key vault](https://docs.microsoft.com/azure/key-vault/general/key-vault-recovery#list-recover-or-purge-a-soft-deleted-key-vault).
+
+### Solution deployment parameters
+
+| Parameter | Type | Description | Default |
+|---|---|---|--|
+|location|string|The Azure Region to Use|[resourceGroup().location]|
```

Comment:
Use > use

Reply:
This is autogenerated from the JSON, I think I've got things consistent now for both the JSON and the README.